tests/helpers/setup-destroy-ei-app.js

```diff
@@ -22,7 +22,10 @@ export async function setupEIApp(EmberDebug, routes) {
     Router.map(routes);
   }
 
-  let App = Application.create({ autoboot: false });
+  let App = Application.create({
+    autoboot: false,
+    rootElement: document.getElementById('ember-testing'),
```

Comment:
Normally this is done in `config/environment.js`, help me grok why we don't want that here?

Reply:
@rwjblue there might be a better way to do it but I’m not entirely sure how. This is a completely manual setup helper (essentially replicating what setupApplicationTest does) that is used in ember-debug tests. It imports directly from @ember/application and doesn’t use the app in either app.js or test-helper.js. It is to simulate a “user app” similar to the ember-source tests. Presumably if we put this in the config I will still have to thread it through here? (Previously it was rendering into document.body, I noticed that and that the test div is more appropriate, and it makes it work with the dom helpers)

Reply:
Gotcha, thank you!